images/image-sign-scan-base/bin/scan-image

```diff
@@ -15,4 +15,4 @@ DOCKERCFG_SECRET_NAME=$(echo "${SA_JSON}" | jq -r ".imagePullSecrets[] | select(
 
 oc get secret -n $NAMESPACE $DOCKERCFG_SECRET_NAME -o json | jq -r ".data[] | select(\".dockercfg\")" | base64 -d > $DOCKER_CFG_HOME/.dockercfg
 
-image-inspector -chroot --path=${IMAGE_CONTENT} --scan-results-dir=${RESULTS} --scan-type=openscap --openscap-html-report --dockercfg=$DOCKER_CFG_HOME/.dockercfg --image=$IMAGE -serve=0.0.0.0:8080
\ No newline at end of file
+image-inspector -chroot --path=${IMAGE_CONTENT} --scan-results-dir=${RESULTS} --scan-type=openscap  --dockercfg=$DOCKER_CFG_HOME/.dockercfg --image=$IMAGE -serve=0.0.0.0:8080
```

Comment:
The html report should still be generated to support the CI/CD integration.

Reply:
There is actually a bug in image-inspector when generating HTML reports. See https://github.com/openshift/image-inspector/issues/105
Not sure how to address this until the PR gets merged besides taking the HTML generation out.

Reply:
This potentially might have been why I initially went down the custom image route with the RPM instead of the OCP image.

I am on the fence on which to move forward with now because the lack of html reports is a big loss in the value that is provided by this solution that can be shown to management

Reply:
I agree, it would be a shame to lose. I'l compare the RPM distributed version of image-inspector with the container-provided one.

Reply:
The image tested in the PR is includes `image-inspector-2.4.0-3.el7.x86_64`

Reply:
The image tested in the PR is `openshift3/image-inspector:v3.11.51-3` includes `image-inspector-2.4.0-3.el7.x86_64`
The previous image-sign-scan-base installs the latest copy from the `rhel-7-server-ose-3.11-rpms`  or whatever version of OCP being used. (in the previous case v3.9) The package version for 3.9 is `image-inspector-2.1.2-2.el7.x86_64` which doesn't suffer from the above bug. However it is missing crucial header handling which breaks the retrieval of the CVE data since a recent update on the RH servers hosting it. Either way, it isn't working.

Reply:
Fun! I just went ahead and tagged the repository as it stands today to `v1.0.0` so we have a way to utilize somewhat working functionality. Would you be able to document what functionality will be lost due to the associated bugs? Otherwise, we should be good to move forward.

Reply:
I added an additional Known issues section